builder/scaleway/config.go

```diff
@@ -57,7 +57,11 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	c.UserAgent = useragent.String()
 
 	if c.Organization == "" {
-		c.Organization = os.Getenv("SCALEWAY_API_ACCESS_KEY")
+		if os.Getenv("SCALEWAY_ORG") != "" {
+			c.Organization = os.Getenv("SCALEWAY_ORG")
+		} else {
+			c.Organization = os.Getenv("SCALEWAY_API_ACCESS_KEY")
```

Comment:
What do you think about logging a deprecation warning here? It gives us the option of removing the extra logic at some point in the future.

Reply:
No, it's basically manual at this point. Thanks for adding the warning.